index.html

```diff
@@ -4,9 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Restaurant One Page HTML5 Template</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
 </head>
 <body>
-    
 </body>
+<script src="node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
```

Comment:
Cette ligne n'est plus utile

Reply:
This line actually injects the bootstrap script into the page, it is indeed useful